### ISSUE TITLE
Aws sdk v2 will enter maintenance mode 

### ISSUE THREAD

**Issue by axmad386** [kodepandai/flydrive#1]
Warn from aws sdk:
```
(node:41155) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
```
I think we should migrate to aws sdk v3

**Comment by axmad386** [comment]
Implement in branch v3 because it has breaking change. Will release it soon

**Comment by axmad386** [comment]
published to v3.0.0-next
